bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_rooabscollection.py

```diff
@@ -33,7 +33,20 @@ def addClone(self, arg, silent=False):
         The keywords must correspond to the CmdArgs of the function.
         """
         clonedArg = self._addClone(arg, silent)
-        SetOwnership(clonedArg, False)
+        # There are two overloads of RooAbsCollection::addClone():
+        #
+        #   - RooAbsArg *addClone(const RooAbsArg& var, bool silent=false);
+        #   - void addClone(const RooAbsCollection& list, bool silent=false);
+        #
+        # In the case of the RooAbsArg overload, we need to tell Python that it
+        # doesn't own the returned pointer. That's because the function name
+        # contains "Clone", which makes cppyy guess that the returned pointer
+        # points to a clone owned by the caller. In the case of the
+        # RooAbsCollection input, the return value will be `None` and we don't
+        # need to change any ownership flags (in fact, calling
+        # SetOwnership(None, False) would cause a crash).
+        if not clonedArg is None:
```

Comment:
For readability, it is preferred `if X is not None` rather than `if not X is None`, see:
https://peps.python.org/pep-0008/#programming-recommendations

Reply:
Okay, you're right, thanks! I have changed this.